firefox_ui_tests/remote/security/test_safe_browsing_notification.py

```diff
@@ -0,0 +1,120 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette_driver import (
+    By,
+    expected
+)
```

Comment:
With only two imports lets keep it on a single line like we do in all the other tests.